pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/AggregationFunctionFactory.java

```diff
@@ -137,6 +137,8 @@ public static AggregationFunction getAggregationFunction(AggregationInfo aggrega
             return new FastHLLAggregationFunction(column);
           case DISTINCTCOUNTTHETASKETCH:
             return new DistinctCountThetaSketchAggregationFunction(arguments);
+          case DISTINCTCOUNTRAWTHETASKETCH:
```

Comment:
this should not be called distinctCount right, you can do anything with thetasketch. maybe just RAWTHETASKETCH

Reply:
Agree, but following the convention of the distinctCountRawHLL. Perhaphs we can alias them both later.